client/web/public/index.html

```diff
@@ -16,9 +16,9 @@
 	<title>Graphite</title>
 </head>
 
-<body>
+<body oncontextmenu="return false">
```

Comment:
Remove this attribute and add the following right below the import statements to `main.ts`:

```js
document.addEventListener("contextmenu", (e) => e.preventDefault());
```